### ISSUE TITLE
Can I use this code to extract pose from still image without any key point? 

### ISSUE THREAD

**Issue by sohayousefi** [yuanyuanli85/Stacked_Hourglass_Network_Keras#8]
Hi 
Can I use this code to extract pose from still image without any key point? Because I want extract pose from Pascal voc2012 but there aren't any key point in this dataset. can you help me how I can extract pose from this dataset please?

**Comment by yuanyuanli85** [comment]
yes, you can refer to 'src/top/demo.py' for how to run inference to extract pose from still image.
